homeassistant/components/knx/storage/entity_store_schema.py

```diff
@@ -87,24 +85,6 @@
 )
 
 
-def optional_ga_schema(key: str, ga_selector: GASelector) -> VolDictType:
-    """Validate group address schema or remove key if no address is set."""
-    # frontend will return {key: {"write": None, "state": None}} for unused GA sets
```

Comment:
Frontend doesn't do this anymore, but omits empty keys. So this isn't needed anymore.